authN-authZ/auth-apisix/keycloak_install.yaml

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2024 Intel Corporation
```

Comment:
Is this a duplicate file of the one sit within the auth-istio folder? If yes, simply refer to the file there.

Reply:
I have moved this file to parent folder for common usage

Reply:
Cool